src/components/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { Fragment } from 'react';
+import Footer from './Footer';
+import Header from './Header';
+import PhotoAlbum from './PhotoAlbum';
+import styles from '../css/App.css';
+
+function App() {
+  return (
+    <Fragment className={styles}> 
```

Comment:
```suggestion
    <div className={styles}> 
```
Stylesheets should not be applied to Fragments (instead, you would use the appropriate HTML tag). The following error will show in the browser:

> react.development.js:188 Warning: Invalid prop `className` supplied to `React.Fragment`. React.Fragment can only have `key` and `children` props.